### PR TITLE
Make more obvious that devs don't need both iOS and Android installed…

### PR DESCRIPTION
**Pull request by sethladd**
… to get started

Didn't want people to think they needed to install the iOS bits to keep following this guide step by step.

### CODE REVIEW

**Comment by sethladd** [comment]
cc @abarth

File: setup.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ nav_title: Flutter Setup
 permalink: /setup/
 ---
 
+(Please remember, Flutter is in _technology preview_. Thanks
+for trying it out, and please don't hesitate to
+[file issues or bugs](https://github.com/flutter/flutter/issues/new)
+for us.)
```
Comment (abarth) [inline setup.md]: Should we have this note on every page?  Maybe we should incorporate it into the banner at the top of the page somehow.

**Comment by abarth** [comment]
LGTM but you probably want the thumbs up from @qchong

File: setup.md (hunk 1 above)
Comment (sethladd) [inline setup.md]: good call. I'll add it to our top banner, right after landing this (and I'll remove this note here)

**Comment by sethladd** [comment]
@qchong ptal !

File: setup.md (hunk 1 above)
Comment (qchong) [inline setup.md]: +1 to a banner label (or icon) to indicate the Tech Preview status of Flutter.

**Comment by qchong** [comment]
lgtm, thanks Seth!

**Pull request merged**